Fix language and typos (#979)
grafana/grafana-infinity-datasource: docs/sources/setup/authentication.md

## No authentication

> If your APIs do not require any authentication, select the **No Authentication** method.

Reviewer: ```suggestion
If your APIs don't require any authentication, select the **No Authentication** method.
```

| **In**    | Accepts `header`/`query`. Most APIs accept API keys via headers as a preferred way of sending API keys. Sending API keys via the query parameter is. |

{{< admonition type="tip" >}}

> It is easy to confuse API key authentication with bearer token authentication. Ensure sure you are using the correct authentication mechanism.

Reviewer: ```suggestion
It's easy to confuse API key authentication with Bearer token authentication.
Ensure sure you are using the correct authentication mechanism.
```
